src/Voron/Debugging/StorageReportGenerator.cs

```diff
@@ -322,8 +322,34 @@ public static TreeReport GetReport(Tree tree, bool includeDetails)
             return treeReport;
         }
 
-        private static StreamsReport CreateStreamsReport(Tree tree)
+        private static StreamsReport CreateStreamsReport(Tree tree, bool includeDetails = false)
         {
+            if (includeDetails == false)
+            {
+                // there are cases we don't need/want to pull the entire data for the report
+                // so we create a report that includes the metadata info
+                // without loading the actual data 
+
+                return new StreamsReport
+                {
+                    AllocatedSpaceInBytes = -1,
+                    NumberOfStreams = tree.State.NumberOfEntries,
+                    Streams = new List<StreamDetails>
```

Comment:
the item inside look fake, do we really need it?

Reply:
It is fake, we need that so the studio will show the entry, because without the details, we are basically lying to the user.

Something to check, we may want to actually report this in inverse. So we figure out how much total space we *aren't* accounting for, and assign it to this value.

Reply:
I think we still need to go over the streams that we have there and calculate `StreamsReport.TotalNumberOfAllocatedPages`. I think that just calculating the total size is fine.  This way we'll be able to properly display the size of `Trees`:

![image](https://user-images.githubusercontent.com/376551/140736102-367d4dac-509c-488a-aa6b-d48945868abf.png)


and  `Attachments` tree here:

![image](https://user-images.githubusercontent.com/376551/140733308-f9e1f770-f29c-4012-a4ae-3f68a3eec071.png)


What we _don't_ need to include are the details of each stream (`StreamDetails`). We need those _only_ if a user goes deeper to see that:

![image](https://user-images.githubusercontent.com/376551/140733993-9d30a541-b2d5-4695-9e35-be97d066683b.png)